tests/integrations/android_sdk/AndroidSDK/test_properties.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
```

Comment:
I don't love the `test_properties` name. I don't have any better ideas. Feel free to leave it as-is :)

Reply:
Yeah - I'm not 100% wild about it either; but it's a convention that I've used elsewhere in the codebase rather than have dozens of 1-test files.